pytket/tests/classical_test.py

```diff
@@ -487,7 +483,7 @@ def test_add_wasm_to_reg() -> None:
 
 def test_wasmfilehandler_without_init() -> None:
     with pytest.raises(ValueError):
-        w = wasm.WasmFileHandler("testfile-without-init.wasm")
+        _ = wasm.WasmFileHandler("testfile-without-init.wasm")
```

Comment:
Maybe you want to to search and replace ` _ = wasm..WasmFileHandler` with `wasm.WasmFileHandler` ?

Reply:
Or we could change the dummy-variable regex in the ruff config: https://docs.astral.sh/ruff/settings/#lint_dummy-variable-rgx

Reply:
Actually I think I'd prefer to allow `_` as a dummy variable since this adheres to Python convention.

Reply:
Does ruff complain about not having this?

Reply:
Ruff is fine with assigning to an unused variable named `_` -- unless you change the default [dummy-variable regex](https://docs.astral.sh/ruff/settings/#lint_dummy-variable-rgx) in the ruff config.